packages/terra-overlay/src/Overlay.jsx

```diff
@@ -79,20 +84,23 @@ class Overlay extends React.Component {
 
   componentWillUnmount() {
     document.removeEventListener('keydown', this.shouldHandleESCKeydown);
-    this.resetBeforeOverlay();
+    this.resetBeforeOverlay(this.props.rootSelector);
   }
 
   setContainer(node) {
     if (!node) { return; } // Ref callbacks happen on mount and unmount, element is null on unmount
     this.overflow = document.documentElement.style.overflow;
+    const selector = this.props.rootSelector;
 
     if (this.props.isRelativeToContainer) {
       this.container = node.parentNode;
       this.containerChildren = this.container.children;
       this.disableContainerChildrenFocus();
```

Comment:
Let's add this to set the container to aria-hidden when using overlay relative to a container.
```suggestion
      this.disableContainerChildrenFocus();
      this.container.setAttribute('aria-hidden', 'true');
```